julia/core.py

```diff
@@ -286,7 +289,11 @@ def __init__(self, init_julia=True, jl_init_path=None):
                     "Install PyCall by running the following line:\n"
                     """\tjulia -e 'Pkg.add("PyCall"); Pkg.update()'\n""")
             try:
-                self.call('pyinitialize(C_NULL)')
+                if os.name == "nt":
+                    # on windows C_NULL doesn't mean anything
+                    self.call('pyinitialize()')
```

Comment:
This doesn't seem right to me.  Running `pyinitialize()` tries to find `libpython` by running the `python` executable.  If you are initializing PyCall from *within* Python, you want it to use the `libpython` that is already loaded into memory.

If you pass `C_NULL` to `pyinitialize`, it initializes `libpython` [by special code](https://github.com/stevengj/PyCall.jl/blob/183c915804ae470cb0a6acff123551dd27dde43d/src/pyinit.jl#L135) that [loads the base module](https://github.com/JuliaLang/julia/blob/233ebd85552345bc550d112480688680d7b0d8f5/src/dlload.c#L87-L97).